modules/formulize/class/elementrenderer.php

```diff
@@ -1042,7 +1042,7 @@ function constructElement($form_ele_id, $ele_value, $entry, $isDisabled=false, $
 			break;
 
 
-			case 'upload':
+			case 'fileUpload':
```

Comment:
I was checking the multi page one. The file upload didn't get saved because when the value of < intput type='file'...> changed, it didn't set the flag to 1 so that when we change page, the page will not know there is something changed, therefore, it's not saved. 

everything else got saved because they have a  condiction which is onchange="javascript:formulizechanged=1;"

in the javascript code, when we changing the page, it will check this var, if true then save.
That explains when i was having two elements when i change both, file gets uploaded. 

But you made a good point, i don't know where is this fileuploadelement.php get called, but it's better we add something like onchange="javascript:formulizechanged=1;" too.